torchci/lib/bot/drciBot.ts

```diff
@@ -0,0 +1,74 @@
+import { Context, Probot } from "probot";
+
+const drciCommentStart = "<!-- drci-comment-start -->";
+const drciCommentEnd = "<!-- drci-comment-end -->";
+
+async function getDrciComment(
+  context: Context,
+  prNum: number,
+  owner: string,
+  repo: string
+): Promise<[number, string]> {
+  const commentsRes = await context.octokit.issues.listComments({
+    owner,
+    repo,
+    issue_number: prNum,
+  });
+  for (const comment of commentsRes.data) {
+    if (comment.body!.includes(drciCommentStart)) {
+      return [comment.id, comment.body!];
+    }
+  }
+  return [0, ""];
+}
+
+export function formDrciComment(): string {
+  let body = "hello there!"
+  return drciCommentStart + body + drciCommentEnd;
+}
+
+export default function drciBot(app: Probot): void {
+  app.on(["pull_request.opened", "pull_request.synchronize"], async (context) => {
+    const prNum = context.payload.pull_request.number;
+    const pr_owner = context.payload.pull_request.user.login;
+    const repo = context.payload.repository.name;
+    const owner = context.payload.repository.owner.login;
+    
+    console.log(pr_owner);
+    if (pr_owner == "swang392") {
```

Comment:
nit: Might be good to have a list here and just check if the user is in the list.

Would make it easier for when you want to expand this out to more users